gallery/visualtests/nxBinaryDonutChart.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { simpleTest } = require('./testUtils');
+
+describe('NxBinaryDonutChart', function() {
+  beforeEach(async function() {
+    await browser.url('#/pages/NxBinaryDonutChart');
+  });
+
+  const simpleDonutSelector = 'nx-binary-donut-chart-examples';
```

Comment:
I doubt this is working.
```suggestion
  const simpleDonutSelector = '#nx-binary-donut-chart-examples';
```

Reply:
> I doubt this is working.

I added a visual test for each example we have for the gallery b9f8078b190bac0c253d7a7c363ad492ea6a5587